share/spice/launch_library/launch_library.js

```diff
@@ -0,0 +1,72 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_launch_library = function(api_result){
+
+        // If the response failed on the server, don't render the Instant Answer at all
+        if (!api_result || api_result.status == "fail" || api_result.total === undefined) {
+            return Spice.failed('launch_library');
+        }
+
+        if (api_result.total > 0) {
+            DDG.require("moment.js", function() {
+                Spice.add({
+                    id: "launch_library",
+
+                    name: "Answer",
+                    data: api_result.launches,
+                    meta: {
+                        itemType: (api_result.number === 1 ? "Upcoming Launch" : "Upcoming Launches"),
+                        sourceName: "LaunchLibrary.net",
+                        sourceUrl: 'http://launchlibrary.net/'
+                    },
+                    normalize: function(item) {
+                        return {
+                            title: item.name,
+                            subtitle: moment(item.net).format("MMM DD, YYYY"),
+                            url: item.vidURL,
+                            description: "Launching from " + item.location.pads[0].name
+                            
+                        };
+                    },
+                    templates: {
+                        group: 'text',
+                        detail: false,
+                        item_detail: false,
+                        options: {
+                            footer: Spice.launch_library.footer,
+                            moreAt: true
+                        }
+                    },
+                    variants: {
```

Comment:
Whoops, this actually needs to be inside the `templates` hash above. Once it's moved, the tiles will look **much** better :+1: